Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM jetbrains/teamcity-agent
+
+RUN apt-get update && \
+  apt-get install curl sudo -y && \
+  curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - && \
```

Comment:
Why is this CURL statement/script needed? I have never done anything like that to install NodeJS and NPM before?

Reply:
I did this to make sure I install the (current) latest LTS version of node. Using apt-get I would get 4.X which is part of Ubuntu LTS right? 

Is there a better way I could do this, as an alternative to using curl?

Also, using apt-get we would have create a `node` symlink to `/usr/bin/node/` to avoid the `nodejs` issue.

Reply:
I think there is a package called "nodejs-legacy" or something similar that installs the symlink too. But I think its fine the way it is now :)